### PR TITLE
en: add what's new in v1.2 

### PR DESCRIPTION
**Pull request by qiancai**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1329
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/whats-new-in-v1.2.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: What's New in TiDB Operator 1.2
+---
+
+# What's New in TiDB Operator 1.2
+
+TiDB Operator 1.2 introduces the following key features, which helps you manage TiDB clusters and tools more easily in terms of extensibility, usability, and security.
```
Comment (qiancai) [inline en/whats-new-in-v1.2.md:7]:
```suggestion
TiDB Operator 1.2 introduces the following key features, which help you manage TiDB clusters and tools more easily in terms of extensibility, usability, and security.
```

File: en/whats-new-in-v1.2.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: What's New in TiDB Operator 1.2
+---
+
+# What's New in TiDB Operator 1.2
+
+TiDB Operator 1.2 introduces the following key features, which help you manage TiDB clusters and tools more easily in terms of extensibility, usability, and security.
```
Comment (TomShawn) [inline en/whats-new-in-v1.2.md:7]:
```suggestion
TiDB Operator 1.2 introduces the following key features, which helps you manage TiDB clusters and the tools more easily in terms of extensibility, usability, and security.
```

File: en/whats-new-in-v1.2.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+- Support customizing labels and annotations for Pods and services in TidbCluster
+- Support configuring podSecurityContext and topologySpreadConstraints for all TiDB components
+- Support full lifecycle management of [Pump](https://docs.pingcap.com/tidb/stable/tidb-binlog-overview#pump)
+- TidbMonitor supports [monitoring multiple TidbClusters](monitor-a-tidb-cluster.md#monitor-multiple-clusters)
+- TidbMonitor supports remotewrite
+- TidbMonitor supports [configuring Thanos sidecar](aggregate-multiple-cluster-monitor-data.md)
+- TidbMonitor management resource changes from Deployment to StatefulSet to provide more flexibility
+- Support [deploying TiDB Operator with namespace permissions only](deploy-tidb-operator.md#customize-tidb-operator-deployment)
+- Support [deploying multiple TiDB Operators](deploy-multiple-tidb-operator.md) to manage different TiDB clusters separately
```
Comment (TomShawn) [inline en/whats-new-in-v1.2.md:19]:
```suggestion
- Support [deploying multiple sets of TiDB Operator](deploy-multiple-tidb-operator.md) to manage different TiDB clusters separately
```

File: en/whats-new-in-v1.2.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Usability
+
+- Add [DMCluster CR to manage DM 2.0](deploy-tidb-dm.md)
+- Support customizing store labels for TiKV and TiFlash
+- Support [customizing storage for TiDB slow log](configure-a-tidb-cluster.md#configure-pv-for-tidb-slow-logs)
+- TiDB Lightning chart supports [configuring local backend and persisting the checkpoint information](restore-data-using-tidb-lightning.md)
```
Comment (TomShawn) [inline en/whats-new-in-v1.2.md:26]:
```suggestion
- TiDB Lightning chart supports [configuring local-backend and persisting the checkpoint information](restore-data-using-tidb-lightning.md)
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
Minor comments. Rest LGTM

**Pull request merged**